collectors/python.d.plugin/unbound/README.md

```diff
@@ -0,0 +1,114 @@
+# Unbound monitoring with Netdata
```

Comment:
I don't think this file is supposed to exist any more! Not sure how it ended up here. @shortpatti, I think you can delete this on your branch.

Reply:
It's been removed :)